samples/basic/fade_led/src/main.c

```diff
@@ -46,9 +46,9 @@ void main(void)
 
 	printk("PWM-based LED fade\n");
 
-	pwm = device_get_binding(PWM_LABEL);
-	if (!pwm) {
-		printk("Error: didn't find %s device\n", PWM_LABEL);
+	pwm = DEVICE_DT_GET(PWM_CTLR);
+	if (!device_is_ready(pwm)) {
+		printk("Error: PWM devices is not ready\n");
```

Comment:
Same comment about printing the device (and `s/devices/device/`) applies here and elsewhere in the PR.